bitnami/contour/values.yaml

```diff
@@ -1289,20 +1289,18 @@ ingress:
 
 ## @section Metrics parameters
 ##
-
-## Prometheus Operator service monitors
-## @param metrics.serviceMonitor.namespace Specify if the servicemonitors will be deployed into a different namespace (blank deploys into same namespace as chart)
-## @param metrics.serviceMonitor.enabled Specify if a servicemonitor will be deployed for prometheus-operator.
-## @param metrics.serviceMonitor.jobLabel Specify the jobLabel to use for the prometheus-operator
-## @param metrics.serviceMonitor.interval Specify the scrape interval if not specified use default prometheus scrapeIntervall, the Prometheus default scrape interval is used.
-## @param metrics.serviceMonitor.metricRelabelings [array] Specify additional relabeling of metrics.
-## @param metrics.serviceMonitor.relabelings [array] Specify general relabeling.
-## @param metrics.serviceMonitor.honorLabels Specify honorLabels parameter to add the scrape endpoint
-## @param metrics.serviceMonitor.scrapeTimeout The timeout after which the scrape is ended
-## @param metrics.serviceMonitor.selector Specify honorLabels parameter to add the scrape endpoint
-## @param metrics.serviceMonitor.labels Extra labels for the ServiceMonitor
-##
 metrics:
+  ## Prometheus Operator service monitors
```

Comment:
Could you follow the same format we use in other charts (e.g. Apache) for this kind of object? See
https://github.com/bitnami/charts/blob/main/bitnami/apache/values.yaml#L656